src/transforms/branch.ts

```diff
@@ -1,18 +1,27 @@
 import { getJiraId } from "../jira/util/id";
-import { Context } from "probot/lib/context";
 import issueKeyParser from "jira-issue-key-parser";
 import { getJiraAuthor } from "../util/jira";
 import _ from "lodash";
+import { WebhookPayloadCreate } from "@octokit/webhooks";
+import { GitHubAPI } from "probot";
 
-async function getLastCommit(context: Context, issueKeys: string[]) {
-	const { github, payload: { ref } } = context;
+async function getLastCommit(github: GitHubAPI, webhookPayload: WebhookPayloadCreate, issueKeys: string[]) {
 
 	const {
 		data: { object: { sha } }
-	} = await github.git.getRef(context.repo({ ref: `heads/${ref}` }));
+	} = await github.git.getRef({
+		owner: webhookPayload.repository.owner.login,
+		repo: webhookPayload.repository.name,
+		ref: `heads/${webhookPayload.ref}`
```

Comment:
pulling the `owner` and `repo` out of the payload instead of using this evil `context` object